requirements.txt

```diff
@@ -1,2 +1,2 @@
-pelican==3.5.0
+pelican==3.7.1
```

Comment:
I guess this change wasn't required. Can you please remove this from the pull request?

Reply:
Refer #188

Reply:
@vipulgupta2048 let's work this out in a separate PR as it would be easier to test things with the new dependency.